libs/cart/src/drivers/interfaces/cart-payment-service.interface.ts

```diff
@@ -0,0 +1,13 @@
+import { InjectionToken } from '@angular/core';
+import { DaffCartPaymentMethod } from '../../models/cart-payment';
+
+export interface DaffCartPaymentServiceInterface<T extends DaffCartPaymentMethod> {
+	get(): T;
+	add(payment: T): T;
+	update(id: string | number, payment: T): T;
+	delete(id?: string | number): void;
+}
```

Comment:
Shouldn't `get()` be called with a cartId?

I would expect `add` to return a cart. Otherwise, we'll need to edit the cart in state with the PaymentMethod rather than just getting the entire cart back and replacing the whole cart state (or make a separate call). Seems more correct to let the backend do all the editing of the cart object. Same thing with `update()`

Reply:
Yes, added.

Reply:
`add` for a payment method (and the other various `add`s) could return the new cart or the updated object. I have no preference for one over the other.